### PR TITLE
SEP-9 : Add `external_transfer_memo` field to Financial Account Fields

### PR DESCRIPTION
**Pull request by ydag**
Resolving the issue #1451

### CODE REVIEW

**Comment by ydag** [comment]
Hi @JakeUrban,

After this change how do you feel about deprecating `crypto_memo`? I think `memo` can be used both regular bank and crypto transfers.

File: ecosystem/sep-0009.md [hunk 1]
```diff
@@ -109,6 +109,7 @@ applications use fields that are the most familiar, which are often specific to
 | `bank_number`         | string | Number identifying bank in national banking system (routing number in US)      |
 | `bank_phone_number`   | string | Phone number with country code for bank                                        |
 | `bank_branch_number`  | string | Number identifying bank branch                                                 |
+| `memo`                | string | A destination tag/memo used to identify a transaction                          |
```
Comment (Ifropc) [inline ecosystem/sep-0009.md:112]: WDYT about renaming it to `external_transfer_memo`? 
So it's not confused with Stellar's transaction memo

Comment (ydag) [reply ecosystem/sep-0009.md:112]: That sounds good! And what do you think about deprecation of `crypto_memo` in that case? We can use `external_transfer_memo` for both regular bank and crypto transfers, right? @Ifropc @marcinx

Comment (ydag) [reply ecosystem/sep-0009.md]: Cool! It is updated.

**Review by Ifropc — DISMISSED** [review]

**Review by JakeUrban — APPROVED** [review]

**Comment by Ifropc** [comment]
@ydag could you update version and changelog of the SEP-9 file?

**Review by Ifropc — APPROVED** [review]

**Comment by Ifropc** [comment]
Thanks for proposing this change @ydag 🎉

**Pull request merged**